tests/resources/db/latest_schema.sql

```diff
@@ -16,6 +16,15 @@ CREATE TABLE experiments (
 )
 
 
+CREATE TABLE experiment_tags (
+	key VARCHAR(250) NOT NULL, 
+	value VARCHAR(250), 
```

Comment:
If we end up using a larger maximum tag length limit, we should update `latest_schema` accordingly.

Reply:
yup - this is autogen'd so, i will re-run that code